add_progress.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>PROGRESS TRACKER</title>
+	<link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+	<h1>TRACK & ADD YOUR PROGRESS</h1>
+
+	<!--THIS IS THE ADD PROGRESS PAGE, USERS ENTER THE WORKOUT TYPE THEY PLAYED AND DATE, THEY CAN ALSO PRESS ON THE TRACK PROGRESS BUTTON WHICH REDIRECTS THEM TO CHECK OUT THEIR WORKOUTS-->
+
+
+	<div class="login-box">
+		<form action="insertWorkout.php" method="POST">
+
+			<div class="user-box">
+				<!--FIELD FOR USER ID NUMBER-->
+			<label for ='membership_ID'><b>Your ID Number</b></label>
+			<br>
+			<input type="text" id="membership_ID" placeholder="ID" name="membership_ID">
+			</div>
+			<!--SELECT WORKOUT TYPE FROM DROPDOWN MENU-->
+			<label for='workout type'> <b>Workout Muscle</b> </label>
+  			<select name='class' id='workout_type'>
+   				<option value='full body'>Full Body</a>
+   				<option value='chest'>Chest</a>
+    			<option value='abs'>Abs</a>
+    			<option value ='shoulders'>Shoulders</a>
+    			<option value='legs'>Legs</a>
+    			<option value ='biceps'>Biceps</a>
+    			<option value='back'>Back</a>
+  			</select>
+		
+			<br></br>
+		
+			<!--DATE FIELD-->
+			<label for='Date'> <b>Date</b> </label>
+			<input type="date" id="date" name="class start"
+       		value="2020-11-01"
+       		min="2020-11-01" max="2020-12-31">
+       		<!--BUTTON TO ADD WORKOUT-->
+			<br> <a href="insertWorkout.php"  class="btn btn1"> Add </a> </button>
+			<!--BUTTON TO TRACK PROGRESS ON THE NEXT PAGE-->
+			<br> <a href="progressSearch.html"  class="btn btn1"> Track Progress </a> </button>
```

Comment:
New button for track progress correctly leads to the user search page where users are able to search for their workout history using their membership ID